src/Components/PresentationalComponents/Filters/CustomFilters/SelectCustomFilter.js

```diff
@@ -13,6 +13,7 @@ const SelectCustomFilter = ({ filterData, setFilterData, selectProps, options, f
     const handleOnRadioChange = (filterId, optionName) => {
         const optionValue = options.find(item => item.label === optionName).value;
         setFilterData({ ...filterData, [filterId]: optionValue });
+        setOpen(false);
```

Comment:
This is setting it to close every time you select an option. Though its a radio dropdown and you can only select one item, the behavior is different than what it is in the CVEs table. 

This should either match the CVEs table or that tables filter should be updated to match this.

Reply:
I'm more concerned that it interacts the same way as the other single select dropdown in this modal:
![image](https://github.com/user-attachments/assets/48992689-a214-4a36-803f-8c96bc34c7be)

Reply:
Fair enough